### ISSUE TITLE
aria-valid-attr-value possible false positive with aria-controls

### ISSUE THREAD

**Issue by bebebebebe** [dequelabs/axe-core#1463]
Assumption: aria-controls can be used for a collapsible element. While the element is in collapsed state, it could be the collapsed content is not in the DOM. In this case, we can have aria-controls="someId", but no element present with id "someId."
https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions

Expected: The case described above will not lead to an aria-valid-attr-value error.

Actual: The case described above leads to an aria-valid-attr-value error.

Motivation: I'm concerned that the above is a false positive. Alternatively, if my assumption above is incorrect, it would be helpful to know so I can avoid actual errors!

axe-core version: 3.2

**Comment by WilcoFiers** [comment]
Totally agreed, when on a thing with aria-expanded="false" I would think that should be allowed. Thank you for reporting!

**Comment by iamrafan** [comment]
@WilcoFiers We think, it should also be allowed on elements with aria-selected="false". Here is an example pointed out by one of our users: https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

The inactive tabs should not fail this rule when the elements referenced to aria-controls(hidden elements in this example) are removed form the DOM.
![image](https://user-images.githubusercontent.com/22068434/55552592-a120b500-5692-11e9-91eb-4dc70709acbd.png)

**Comment by iamrafan** [comment]
@straker @WilcoFiers:  aria-owns also should be passed when ```aria-expanded="false"```. For example, comboboxes are required to have popups only when expanded.

**Comment by straker** [comment]
@iamrafan could you open a separate issue for that one? I'll take a look at it as well

**Comment by iamrafan** [comment]
@straker Sure, I filed #1524

**Comment by jeankaplansky** [comment]
@straker : what, if any documentation is required to complete this ticket?
